Makefile

```diff
@@ -143,7 +143,7 @@ integrationtest:
 	@echo "$$INTEGRATION_TEST_HELP_INFO"
 else
 integrationtest:
-	hack/make-rules/build.sh edgecore
+	hack/make-rules/build.sh edgecore; \
```

Comment:
could you please provide some materials about this `;`, is this required? :)

Reply:
I think I was mistaken, I thought that `execute.sh` wasn't being run

Reply:
So what's meaning of the `; \` here?

Reply:
I reverted it, thought it was needed to run both commands